cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app_complex/templates/java/${variables.rootPackage}/${variables.component}/logic/api/usecase/UcFind${variables.entityName}Ctos.java.ftl

```diff
@@ -7,6 +7,16 @@ import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 
 import java.util.List;
+<#assign compositeIdVar = false>
+<#list pojo.fields as field>
+	<#if field.type?starts_with("Composite")>
+		<#assign compositeIdVar = true>
+		<#assign compositeIdTypeVar = field.type>
+	</#if>
+</#list>
```

Comment:
missing refactoring

Reply:
Changes done and committed. Re-Review required.